src/nbl/ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -288,7 +288,7 @@ void main()
 	mat2 dUV = mat2(dFdx(UV),dFdy(UV));
 
 	// "The sign of this computation is negated when the value of GL_CLIP_ORIGIN (the clip volume origin, set with glClipControl) is GL_UPPER_LEFT."
-	const bool front = !gl_FrontFacing;
+	const bool front = (!gl_FrontFacing) != (InstData.data[InstanceIndex].determinant < 0.0);
```

Comment:
Could be turned into `gl_FrontFacing != (cameraDeterminant*instanceDeterminant>0.0)`

Reply:
TODO